src/Store/ActionTypes.js

```diff
@@ -12,9 +12,12 @@ export const CHANGE_SYSTEM_CVE_STATUS = 'CHANGE_SYSTEM_CVE_STATUS';
 export const CHANGE_EXPOSED_SYSTEMS_PARAMETERS = 'CHANGE_EXPOSED_SYSTEMS_PARAMETERS';
 export const CHANGE_CVE_LIST_PARAMETERS = 'CHANGE_CVE_LIST_PARAMETERS';
 export const SELECT_CVE = 'SELECT_CVE';
+export const SELECT_SYSTEM_CVE = 'SELECT_SYSTEM_CVE';
 export const SELECT_ENTITIES = 'SELECT_ENTITIES';
 export const SELECT_MULTIPLE_ENTITIES = 'SELECT_MULTIPLE_ENTITIES';
 export const EXPAND_CVE = 'EXPAND_CVE';
 export const EXPAND_ROW = 'EXPAND_ROW'; //TODO unify it with EXPAND_CVE action
+export const EXPAND_SYSTEM_CVE = 'EXPAND_SYSTEM_CVE';
 export const FETCH_INSIGHTS_SYSTEM_RULES = 'FETCH_INSIGHTS_SYSTEM_RULES';
 export const CHANGE_SYSTEMS_PARAMS = 'CHANGE_SYSTEMS_PARAMS';
+export const CHANGE__SYSTEM_CVE_LIST_PARAMETERS = 'CHANGE__SYSTEM_CVE_LIST_PARAMETERS';
```

Comment:
the double `__` was intentional?

Reply:
I was not able to generate the first issue locally. What I did was following:
 1. I changed page to 2.
 2. I filtered (No issue appeared)
 3. I add more filters such as security_filter, impact (No issue appeared).
 Please let me know, if I need to do something different to generate that issue.

Second issue the double __ is unintentionally. Maybe I had a lot of underscores at that time :))